cmd/crowdsec-cli/explain.go

```diff
@@ -172,6 +179,9 @@ func (cli cliExplain) run(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("couldn't create a temporary directory to store cscli explain result: %s", err)
 	}
 	defer func() {
+		if no_clean {
```

Comment:
Could we just avoid generate the defer func instead of returning straight away?

Reply:
However, might be hard to read `!no_clean` instead of just returning ill let you judge it